ngraph/test/runtime/ie/unit_test.manifest

```diff
@@ -177,7 +177,7 @@ arg_min_no_keep_dims_dyn_shape
 onnx_model_one_hot_with_axis
 
 # Dynamic function 'get_shape was called on a descriptor::Tensor with dynamic shape'
-onnx_upsample9_scales_input_nearest_infer
+# onnx_upsample9_scales_input_nearest_infer
```

Comment:
Please remove tests (do not comment) from manifest file if you enable it.

Reply:
Tests are enabled.